BlackNectar/BlackNectar/FilterViewController.swift

```diff
@@ -216,16 +262,21 @@ extension FilterViewController {
     
 }
 
-//MARK: Loads Stores When User Pans
+//MARK: Style Menu Code
 extension FilterViewController {
     
-    func mapView(_ mapView: MKMapView, regionDidChangeAnimated animated: Bool) {
-        
-        let center = mapView.centerCoordinate
+    func styleButtonOn(button: UIButton) {
         
-        LOG.debug("User dragged Map Screen to: \(center)")
+        button.backgroundColor = Colors.fromRGB(red: 235, green: 191, blue: 77)
+        button.titleLabel?.font = UIFont(name: "Oxygen-Bold", size: 16)
+        UIView.transition(with: button, duration: 0.4, options: .transitionCrossDissolve, animations: nil, completion: nil)
+    }
+    
+    func styleButtonOff(button: UIButton) {
         
-        self.loadStoresInMapView(at: center)
+        button.backgroundColor = UIColor.clear
+        button.titleLabel?.font = UIFont(name: "Oxygen-Regular", size: 16)
```

Comment:
These fonts should ideally be in their own class `Fonts`, so  you can quickly load them as variables, and not have to enter the String name here all the time.